pretext/project/__init__.py

```diff
@@ -549,12 +554,11 @@ def parse(
         xml_bytes = _path.read_bytes()
 
         # Determine the version of this project file.
-
         class ProjectVersionOnly(pxml.BaseXmlModel, tag="project"):
             ptx_version: t.Optional[str] = pxml.attr(name="ptx-version")
 
         p_version_only = ProjectVersionOnly.from_xml(xml_bytes)
-        if p_version_only.ptx_version != "2":
+        if p_version_only.ptx_version is not None:
```

Comment:
This was also a recent change.  Don't we want to assume legacy even if `ptx_version="1"`?

Reply:
Legacy project should not have `ptx_version` in them at all -- if they do have a `ptx_version`, then we assume it's a new project format.

Reply:
Fair.  Given the other issue I was having, I think I started considering cases where someone would try to keep the legacy version by specifying a different attribute.